app/scripts/services/utils.srv.js

```diff
@@ -166,6 +166,19 @@ angular.module('dashboard')
         };
 
         /*
+         * @name dashboard.utils.parseHtmlError
+         * @description Parses an Ahjo Meeting API response error and returns error object
+         * @param {Object} response REST error to process.
+         * @returns {Object} included parsed error { 'errorCode': value }
+         */
+        Utils.parseHtmlError = function (error) {
+            var result = CONST.GENERALERROR;
+            var ec = Utils.processAhjoError(error);
```

Comment:
Should add a 2nd `false` argument for `processAhjoError` to avoid displaying UI notes.